src/test/java/uk/gov/companieshouse/api/testdata/service/impl/OfficerAppointmentServiceImplTest.java

```diff
@@ -44,11 +50,15 @@
 
     @Test
     void create() throws DataException {
+        Address mockAddress = new Address();
         CompanySpec spec = new CompanySpec();
         spec.setCompanyNumber(COMPANY_NUMBER);
+        spec.setJurisdiction(Jurisdiction.ENGLAND_WALES);
 
         when(randomService.getEtag()).thenReturn(ETAG);
         OfficerAppointment savedOfficerAppointment = new OfficerAppointment();
+        when(addressService.getCountryOfResidence(spec.getJurisdiction())).thenReturn("Wales");
```

Comment:
I think we could store "Wales" in a variable to be sent to `assertOfficerItem` so to ensure we are checking the right value? 
We could then even use a different string value to make sure "Wales" (or any other valid jurisdiction) is not actually hardcoded in the service somehow but uses the returned value of the address service